src/components/portal/portal.tsx

```diff
@@ -120,6 +120,13 @@ export class Portal {
             return;
         }
 
+        if (this.containerId) {
+            // eslint-disable-next-line no-console
+            console.warn(
+                'The `containerId` prop is no longer needed. It is deprecated and will be removed in a future release.',
```

Comment:
there is only [one instance](https://github.com/search?q=org%3ALundalogik+%3Climel-portal&type=code) that someone uses `limel-portal`. I'd love to remove this `@private` prop, instead of deprecating it. But I guess this is more by the book.

Reply:
I'm not sure what you mean, you want to make the component public?
Not sure how they found out about this private component :smile:

Reply:
No no. I just mean when components are private, we don't need to deprecate anything. We can just fix the whatever we have removed, so that the consumer components of that deprecated thing keep working as they should.

Now in this case, with my last fixup I'd have loved to remove the `containerID` prop. But since someone is using `limel-portal` in their project, then we will break their shit. So we can't just remove the prop. However, we can add a console warning instead and hope they see it. 😅

Reply:
I was just lazy and deprecated it instead of updating all the code that used it :smile: If you now have removed all those usages, we can just remove it instead :smile:

Reply:
I see 😝 
Yes. I've fixed them all. Pushed an update fixup